client/main.go

```diff
@@ -324,6 +325,17 @@ func main() {
 	}
 
 	flag.Parse()
+	
+	latencyunit := 1000000
```

Comment:
Instead of having two variables that only differ in casing, I would rename this to `latencyDivisor`, and declare it as an `int64` which will clean up using it below.